docs/docs-sidebar.json

```diff
@@ -384,5 +384,19 @@
       "modules/tile-converter/cli-reference/slpk-extractor",
       "modules/tile-converter/cli-reference/supported-features"
     ]
+  },
+  {
+    "type": "category",
+    "label": "Receipts",
```

Comment:
The choice of the word "Receipts" is surprising / confusing to me. 

Why did you pick it? What are we trying to say?

Reply:
I quess, I've mixed it up with "Recipes". These are tutorials for particular use cases.
Eg. 
* https://nx.dev/recipes
* https://docs.cypress.io/examples/recipes